docs/api-docs/storefront/widgets/widget-builder.md

```diff
@@ -87,13 +84,31 @@ To get started, run the following command:
 widget-builder init
 ```
 
-When prompted, enter your API account credentials.
+When prompted, enter your API account credentials. This will create the `.env` file with the necessary parameter assignments.
 
-## Creating a new project
+**Sample output:**
 
-To create a new project, run the following command:
+```shell
+Thank you for using Widget Builder
+
+            
+This guide will help you get your enviornment set up.
+
+Before continuing, please make sure you've created or have been provided a Store API account.
```

Comment:
Before continuing, please make sure you've created or received a Store API account.